Cargo.toml

```diff
@@ -41,6 +41,7 @@ serde_qs = "0.5.0"
 async-std = { version = "1.0.1", features = ["unstable"] }
 mime = "0.3.14"
 cookie = { version="0.12.0", features = ["percent-encode"]}
+simd-json = { version = "0.3.2", optional = true }
```

Comment:
If `simd-json` is enabled, shouldn`t `serde_json` be disabled?

Reply:
serde_json is still used for encoding data, given the performance gains will be minimal compared to decoding we've not gotten around to that part :)

Reply:
I took your inspiration here and added a serializer to simd-json, I stand corrected there is a decent performance benefit in that :)

Reply:
Oh, so even for serializing there is a decent performance benefit over `serde-json` too?

Reply:
Ja here is a run of json-benchmark w/ simd-json serialisation:
```
                                DOM                  STRUCT
======= serde_json ======= parse|stringify ===== parse|stringify ====
data/canada.json         250 MB/s   380 MB/s   510 MB/s   310 MB/s
data/citm_catalog.json   400 MB/s   360 MB/s   930 MB/s   430 MB/s
data/twitter.json        290 MB/s   530 MB/s   590 MB/s   600 MB/s
data/log.json            360 MB/s   720 MB/s   720 MB/s   720 MB/s

======= simd_json ======== parse|stringify ===== parse|stringify ====
data/canada.json         510 MB/s   440 MB/s   680 MB/s   420 MB/s
data/citm_catalog.json  1030 MB/s   550 MB/s  1470 MB/s   460 MB/s
data/twitter.json        910 MB/s   750 MB/s  1050 MB/s   640 MB/s
data/log.json           1080 MB/s  1080 MB/s  1080 MB/s  1080 MB/s

===== simd_json_tape ===== parse|stringify ===== parse|stringify ====
data/canada.json         750 MB/s                    
data/citm_catalog.json  1790 MB/s                    
data/twitter.json       1620 MB/s                    
data/log.json           2170 MB/s                    
```

Reply:
And looking at the results I really want to make a custom derive version for `simd-json`, it seems that `serde-derive::Serialize` does degrade performance things down given that serializing a DOM object is faster.

Reply:
Ah, that's interesting. It seemed like DOM is faster than struct for some of the `simd-json` benchmark.

Reply:
Yap the DOM serialization bypasses the traits and serialized directly.

Reply:
By the way, why not add it to actix too or the techempower benchmarks, I believe it can improve the JSON serialization ranks. IIRC last time actix was the first in JSON serialization now it dropped to 10th.

https://www.techempower.com/benchmarks/#section=test&runid=c7013ea3-22c9-44c1-864f-949509d78e0c&hw=ph&test=json
https://github.com/TechEmpower/FrameworkBenchmarks/blob/cfbbf5d11143c96851620eb1aeaf3c2893e862c9/frameworks/Rust/actix/src/main.rs#L23

Reply:
Looking at that code I don't think it'd make a big difference it'd make a bigger difference not to use struct serialization altogether, the message is tiny so it'd not get much of a benefit from SIMD instructions. I don't want to completely generalize it but at least for simd-json, it'd never get to the SIMD part of the code as the key and value are shorter than a single register. Might Still worth looking into from the perspective of avoiding the Serialize train by using a DOM object instead.

Reply:
I created a tracking issue on the serializer: https://github.com/simd-lite/simd-json/issues/121

Reply:
How about the other link for the techempower benchmark?

Reply:
Since it's not really related and to not spam this PR with it I created https://github.com/simd-lite/simd-json/issues/122 for that topic